utils/prometheus/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Jeffery To
```

Comment:
Can replace my name with yours :smiley:

Reply:
Oops, removed. I don't intent to have any copyright on this...